packages/dapi/scripts/api.js

```diff
@@ -107,17 +107,16 @@ async function main() {
     tenderdashLogger.warn(`Connection closed: ${error.code}`);
   });
 
-  tenderDashWsClient.on('disconnect', () => {
-    tenderdashLogger.fatal('Disconnected from Tenderdash... exiting');
+  const blockchainListener = new BlockchainListener(tenderDashWsClient);
+
+  tenderDashWsClient.connect().then(() => {
+    blockchainListener.start();
+  }).catch((e) => {
+    tenderdashLogger.fatal(`Not able to connect to tenderdash: ${e}`);
 
     process.exit(1);
```

Comment:
We still exit in case cannot connect to tenderdash?

Reply:
It's not gonna happen with current setting `maxRetry: -1` but I would like to have an error handling here anyways. For example if we have some syntax error or other problem.

Reply:
So users still not able to query DAPI until tenderdash actually starts?